insights/parsers/package_provides_httpd.py

```diff
@@ -0,0 +1,63 @@
+"""
+PackageProvidesHttpd - command ``/bin/echo {httpd_command_package}``
+====================================================================
+
+This command reads the output of the pre-command:
+
+``for jp in `/bin/ps auxwww | grep httpd | grep -v grep| awk '{print $11}' | sort -u`; do echo $jp | xargs rpm -qf`; done``
+
+This command looks for all versions of 'httpd' running and tries to find the
+RPM packages which provide them.  The running command and its package name
+are stored as properties ``command`` and ``package`` of the object.
+
+The reason why using above pre_command is that we need to record the links
+between running_httpd_command and package which provides the httpd command. In
+``ps aux`` output, we can only get what httpd command starts a httpd
+application, instead of httpd package. Through this way, when there is httpd
+bug, we can detect whether a running httpd application will be affected.
+
+Typical contents of the pre_command::
+
+    /usr/sbin/httpd httpd-2.4.6-88.el7.x86_64
+
+Parsed result::
+
+    self.command = '/usr/sbin/httpd'
+    self.package = 'httpd-2.4.6-88.el7.x86_64'
+
+Examples:
+    >>> from insights.tests import context_wrap
+    >>> PACKAGE_COMMAND_MATCH = "/usr/sbin/httpd httpd-2.4.6-88.el7.x86_64"
+    >>> command_package = PackageProvidesHttpd(context_wrap(PACKAGE_COMMAND_MATCH))
```

Comment:
Please incorporate the `context_wrap` and input lines in your doctests instead.  Make sure that your examples match the output based on the values above in "Parsed result" so that it will make sense to a developer reviewing the docs and the code.